### PR TITLE
Added Idris Olubisi.

### PR DESCRIPTION
**Pull request by Olanetsoft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,5 +26,5 @@ Any questions let us know!
...
.../22374829/104178614-a8acea80-5430-11eb-8ee8-1a7bb89248b2.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/237">(:100: give your vote)</a></p></td><td align="center"><p><a href="https://github.com/filippo-fonseca">Filippo Fonseca</a></p><img src="https://user-images.githubusercontent.com/48065878/104632919-feb5a400-5663-11eb-8dd0-b05cfd377158.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/244">(:100: give your vote)</a></p></td><td align="center"><p><a href="https://github.com/paulsonjpaul">Paulson J Paul</a></p><img src="https://user-images.githubusercontent.com/73777251/104731775-360a7a80-5762-11eb-8a86-b50d98b58b01.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/247">(:100: give your vote)</a></p></td></tr><tr><td align="center"><p><a href="https://github.com/Olanetsoft">Idris Olubisi</a></p><img src="https://user-images.githubusercontent.com/45847909/104912483-9da40e00-598c-11eb-8881-971faeaecd9d.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/254">(:100: give your vote)</a></p></td><td></td><td></td></tr></table>
```
Comment (AllanRegush) [inline README.md:29]: The action will create this Tables for you. Please revert this change 😄

**Review by AllanRegush — COMMENTED** [review]
Looking Good 👍

**Review by w3bdesign — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AllanRegush) [reply README.md:29]: I seen that the action ran on your fork. I've made a PR to fix this issue.

File: data.json [hunk 2]
```diff
@@ -466,5 +466,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/73777251/104731775-360a7a80-5762-11eb-8a86-b50d98b58b01.png",
     "issueNumber": 247
-  }
+  },
+  {
+    "name": "Idris Olubisi",
+    "githubUsername": "Olanetsoft",
+    "imageUrl": "https://user-images.githubusercontent.com/45847909/104912483-9da40e00-598c-11eb-8881-971faeaecd9d.png",
+    "issueNumber": 254
+}
```
Comment (AllanRegush) [inline data.json:475]: I think this `}` should be lined up like the others.

**Review by w3bdesign — CHANGES_REQUESTED** [review]
This file doesn't need to be modified as the Github action will automatically generate it.

You only need to modify data.json

File: README.md (hunk 1 above)
Comment (Olanetsoft) [reply README.md:29]: Thank you @AllanRegush

File: data.json (hunk 2 above)
Comment (Olanetsoft) [reply data.json]: @AllanRegush fixed

**Comment by Olanetsoft** [comment]
To the best of my knowledge, I didn't modify the ReadME file. @w3bdesign

**Comment by w3bdesign** [comment]
> 
> 
> To the best of my knowledge, I didn't modify the ReadME file. @w3bdesign

Under "Files changed" I see README.md and data.json changed.

**Review by AllanRegush — APPROVED** [review]
Thank you :+1: Looks Good :smile:
